xonsh/style_tools.py

```diff
@@ -342,24 +337,18 @@ def style_as_faded(template: str) -> str:
             Token.Operator.Word: "bold ansimagenta",
             Token.Other: "",
             Token.Punctuation: "",
-            Token.Scrollbar: "bg:ansibrightblack",
-            Token.Scrollbar.Arrow: "bg:ansiblack ansiwhite bold",
-            Token.Scrollbar.Button: "bg:ansiblack",
             Token.Text: "",
             Token.Text.Whitespace: "ansigray",
+            Token.PTK.Aborting: "ansibrightblack",
+            Token.PTK.AutoSuggestion: "ansibrightblack",
+            Token.PTK.CompletionMenu: "bg:ansigray ansiblack",
+            Token.PTK.CompletionMenu.Completion: "",
+            Token.PTK.CompletionMenu.Completion.Current: "bg:ansibrightblack ansiwhite",
+            Token.PTK.Scrollbar.Arrow: "bg:ansiblack ansiwhite bold",
+            Token.PTK.Scrollbar.Background: "bg:ansibrightblack",
+            Token.PTK.Scrollbar.Button: "bg:ansiblack",
```

Comment:
I think these will then need to be added to all styles.

Reply:
They should be there due to change in `pyghooks` where I've replaced the old `PTK_STYLE` to `DEFAULT_STYLE_DICT` in the style chainmap.